krkn/scenario_plugins/node_actions/abstract_node_scenarios.py

```diff
@@ -36,6 +36,16 @@ def helper_node_stop_start_scenario(self, instance_kill_count, node, timeout):
         self.helper_node_start_scenario(instance_kill_count, node, timeout)
         logging.info("helper_node_stop_start_scenario has been successfully injected!")
 
+    # Node scenario to detach and attach the disk
+    def node_disk_detach_attach_scenario(self, instance_kill_count, node, timeout, duration):
+        logging.info("Starting disk_detach_attach_scenario injection")
+        disk_attachment_details = self.get_disk_attachment_info(instance_kill_count, node)
+        self.disk_detach_scenario(instance_kill_count, node, timeout)
+        logging.info("Waiting for %s seconds before attaching the disk" % (duration))
+        time.sleep(duration)
```

Comment:
I'd remove this one from here.

After adding `node_disk_detach_scenario` and `node_disk_attach_scenario` the user can always decide for how long the volume will be detached.

Reply:
Withdraw this comment :/ `node_stop_start_scenario()` does a similar thing (I was misled because "docs/node_scenarios.md" didn't mention the `duration` parameter at all). My bad :(